wiki/UI-component/Video.md

```diff
@@ -31,14 +31,75 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 <details open="true">
   <summary><strong>Resources</strong></summary><br />
 
-- r1: [Box folder](https://ibm.ent.box.com/folder/108837570522)
+- r1: [Box folder](https://ibm.box.com/s/9q7rdj73m32hufcw1xwl1td6nr88rghm)
 - r2: [Kaltura player](http://player.kaltura.com/docs/api)
 
 <br />
 </details>
 
 <hr width="40%" />
 
-<br />
+## Functions
+
+| Variants | Interaction | Functions |
+| -------- | ----------- | --------- |
+| Default | Clicking on `Poster image` | Poster image is replaced with Kaltura video player and video will begin playing. |  
+| with lightbox media viewer | Clicking on `Poster image` | Open LightBoxMediaViewer and video will begin playing. |
+
+
+## Logics
+
+### Default variant
+
+Video allows custom override of all content returned by the [[Video service]] except video duration.
+
+| ID  | Field |
+| --- | ----- |
+| 1   | Caption |
+| 2   | Poster image |
+| 3   | Poster image alt tag |
+
+##### 1. Caption
+
+| Condition | Description |
+| --------- | ----------- |
+| `undefined` | Video name prefilled by [[Video service]]. Duration is appended after name in parenthesis. |
+| Text | Adding a custom caption overrides the API call. Duration is appended after custom caption in parenthesis. |
```

Comment:
I would personally add more of a template of how it's intended instead of being so verbose. Or not describe how it should appear at all since that's where design comes in more. Honestly, your first sentence of each row might be good enough here.

```
video.title (video.duration)
```